site/docs/components/card/examples.mdx

```diff
@@ -18,14 +18,20 @@ By default, the card component does not have any interactive or focusable elemen
 
 </LivePreview>
 
-<LivePreview componentName="card" exampleName="Variants" >
-
 ## Variants
 
+<LivePreview componentName="card" exampleName="Variants" >
+
 All card components provide primary, secondary and tertiary variants, offering flexibility and enabling differentiation based on visual hierarchy within your UI.
```

Comment:
NIT: consistency
on line 31 we use backticks for `ghost` but not here for primary, secondary and tertiary
but all of them are referring to variant values.